src/controllers/company.rs

```diff
@@ -0,0 +1,169 @@
+use crate::models::company::{Company, CompanyId, NewCompany};
+use crate::models::pagination::{Pagination, PaginationMethods};
+use crate::models::store_trait::StoreMethods;
+use crate::service::convert_to_json::{Data, PayloadNoData, PayloadWithData};
+use crate::service::jwt::Claims;
+use std::collections::HashMap;
+use std::sync::Arc;
+use tracing::{event, instrument, Level};
+use warp::http::StatusCode;
+
+// Handle for create company
+//
+// This function adds a new company to the system. It takes company information to
+// be created and a reference to the StoreMethods trait object for company. It
```

Comment:
"It takes company information to be created and a reference to the StoreMethods trait object for company. " No helpful info